client/js/test/fixtures/sheet-container.html

```diff
@@ -9,5 +9,7 @@
     	<tr>
     	</tr>
     </tbody>
+    <tfoot>
+    </tfoot>
 	</table>
```

Comment:
Careful of the indentation. All nested portions should be indented from their parents.